lib/src/chewie_player.dart

```diff
@@ -225,6 +225,7 @@ class ChewieController extends ChangeNotifier {
     this.aspectRatio,
     this.autoInitialize = false,
     this.autoPlay = false,
+    this.dragableProgressBar = true,
```

Comment:
Please fix this typo: draggable...

If you rename this symbol it should be good to go

Reply:
Fixed it https://github.com/brianegan/chewie/pull/498/commits/a350cadb84a3418eaf8765d397929d752717c62a